new-website/deepchem/layouts/tutorial.js

```diff
@@ -0,0 +1,124 @@
+import Script from "next/script";
+import { useEffect, useState } from "react";
+import { useRouter } from "next/router";
+
+import TutorialLink from "../components/Tutorials/TutorialLink";
+import tutorials from "../data/tutorials/tutorials";
+
+import ScrollToTop from "react-scroll-to-top";
+
+/**
+ * Tutorial layout component - A wrapper layout for all the tutorials pages
+ * @component
+ * @param {Object} props - props passed to the component
+ * @param {Object} props.children - children to be rendered inside the Default layout
+ * @return {JSX} - JSX for the TutorialLayout component
+ */
+export default function TutorialLayout({ children }) {
+  const [currentTutorialIndex, setCurrentTutorialIndex] = useState(1);
+  const [isNavbarOpen, setIsNavbarOpen] = useState(false);
+  const router = useRouter();
+
+  /**
+   * Funtion to toggle the tutorials navbar
+   * @function
+   * @return {void}
+   */
+  function toggleNavbar() {
+    setIsNavbarOpen((prev) => !prev);
+  }
+
+  useEffect(() => {
+    if (!router.isReady) return;
+    const url = router.pathname;
+    const tutorialName = url.split("/")[2];
+    const tutorialIndex = tutorials.findIndex((element) =>
+      element.urlifiedFileName.includes(tutorialName)
+    );
+    setCurrentTutorialIndex(tutorialIndex);
+  }, [router.isReady]);
+
+  useEffect(() => {
+    window.addEventListener("resize", () => {
+      if (window.innerWidth > 1024) {
+        setIsNavbarOpen(false);
+      }
+    });
+  }, []);
+
+  return (
+    <>
+      <Script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-AMS_CHTML-full,Safe" />
```

Comment:
Can we add some documentation around the use/support of mathjax?

Reply:
Sure, I'll add some documentation about the mathjax configs used in out implementation

Reply:
@rbharath added the documentation. Sorry for the delay